packages/create-next-app/templates/index.ts

```diff
@@ -94,6 +98,22 @@ export const installTemplate = async ({
 
     await install(root, dependencies, installFlags)
   }
+  /**
+   * Install package.json devDependencies if they exist.
+   */
+  if (devDependencies.length) {
+    console.log()
+    console.log('Installing devDependencies:')
+    for (const devDependency of devDependencies) {
+      console.log(`- ${chalk.cyan(devDependency)}`)
+    }
+    console.log()
+
+    await install(root, devDependencies, {
+      ...installFlags,
+      devDependencies: true,
+    })
+  }
```

Comment:
I just copy-pasted this section for the devDependencies addition, but I wasn't sure if you wanted to have all the install logs together vs in two places.

See the screenshot for an example.

![CleanShot 2022-11-12 at 23 11 54@2x](https://user-images.githubusercontent.com/3606121/201503909-d9fe4732-55d7-47c6-a328-83787acd98de.png)